blocks/bricks/sequence_generators.py

```diff
@@ -440,7 +440,15 @@ class SoftmaxEmitter(AbstractEmitter, Initializable, Random):
 
     Interprets readout elements as energies corresponding to their indices.
 
+    Parameters
+    ----------
+    initial_output : int or a scalar :class:`~theano.Variable`
+        The initial output.
```

Comment:
Although I understand that documentation is insufficient here, the proper place for it would be the `AbstractEmitter` documentation. It also will have to play nicely with all the rest of documentation, which in turn will change significantly after #522. Taking it all into account, I would not touch it here for now.f